hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/s3guard/ITestDynamoDBMetadataStore.java

```diff
@@ -725,50 +975,55 @@ public void testDeleteTable() throws Exception {
       intercept(IOException.class, "",
           "Should have failed after the table is destroyed!",
           () -> ddbms.listChildren(testPath));
-    } finally {
       ddbms.destroy();
-      ddbms.close();
+      intercept(FileNotFoundException.class, "",
+          "Destroyed table should raise FileNotFoundException when pruned",
+          () -> ddbms.prune(0));
+    } finally {
+      destroy(ddbms);
     }
   }
 
+/*
```

Comment:
Remove commented out test, or `@skip` with justification.

Reply:
remove. I'd originally moved the tagging code into the static, then pulled it into the aggregate testTableVersioning which combines 4 tests which were creating and destroying individual tables